macro_utils_h_generator/macro_utils.tt

```diff
@@ -26,290 +26,290 @@
 #include <string.h>
 
 #if (defined OPTIMIZE_RETURN_CODES)
-    #define __FAILURE__ 1
+    #define __MU_FAILURE__ 1
```

Comment:
>__MU_FAILURE__ [](start = 12, length = 14)

While we are here and a rename will have to follow the complete code base, can you please change this to NOT start with 2 underlines? This is what the C standard has to say about identifiers that start with a double score: Chapter 7.1.3 Reserved identifiers "All identifiers that begin with an underscore and either an uppercase letter or another underscore are always reserved for any use." 

That means we cannot have a __MU_FAILURE__ - you can baptize it to whatever else that is not reserved.

Reply:
+1 on this

Reply:
Very fair and right. It shall be renamed to MU_FAILURE